docs/resources/credential.md

```diff
@@ -172,6 +172,39 @@ resource "tanzu-mission-control_credential" "tmc_provisioned_aws_s3_cred" {
 }
 ```
 
+# Credential for Self provisioned AWS S3 or S3-compatible storage used for data-protection
+
+## Example Usage
+
+```terraform
+# Create Self provisioned AWS S3 or S3-compatible credential
+resource "tanzu-mission-control_credential" "aws_eks_cred" {
+  name = "tf-aws-s3-self-test"
+
+  meta {
+    description = "Self provisioned AWS S3 or S3-compatible storage credential for data protection"
+    labels = {
+      "key1" : "value1",
+    }
+  }
+
+  spec {
+    capability = "DATA_PROTECTION"
+    provider   = "GENERIC_S3"
+    data {
+      key_value {
+        type = "OPAQUE_SECRET_TYPE"
+        data = {
+          "aws_access_key_id"          = "abcd="
+          "aws_secret_access_key"      = "xyz=="
+        }
+      }
+    }
+  }
+  ready_wait_timeout = "1m" // Wait time for credential create operations to finish (default: 3m).
```

Comment:
1m is lesser compared to the default time, I assume the crew creation takes more than 1m.

Reply:
It was good enough when I tested, anyways I have increased it to 5m now

Reply:
The change is not reflecting here

Reply:
Please refresh, it is already updated

Reply:
Please regenerate the doc, to reflect the changes